utils/util-linux/DEPENDS

```diff
@@ -3,8 +3,7 @@ depends shadow
 depends zlib
 depends findutils
 depends cpio
+depends libcap-ng
 
-optional_depends python  "--with-python=3" "" "for Python 3 support" y
-optional_depends python2 "--with-python=2" "" "for Python 2 support" n
-
+optional_depends python  "" "" "for Python 3 support" y
```

Comment:
If there is a meson option like `-Dpython=true|false`, please use that in optional_depends, otherwise, it will be impossible to disable python support if it is installed.